ambry-store/src/main/java/com.github.ambry.store/StorageManager.java

```diff
@@ -58,6 +58,8 @@ public StorageManager(StoreConfig config, ScheduledExecutorService scheduler, Me
       MessageStoreHardDelete hardDelete, Time time) throws StoreException {
     verifyConfigs(config);
     metrics = new StorageManagerMetrics(registry);
+    StoreMetrics storeMainMetrics = new StoreMetrics(registry);
+    StoreMetrics storeUnderCompactionMetrics = new StoreMetrics("UnderCompaction", registry);
```

Comment:
why not call it as "compactionMetrics" or "Compactor" instead of storeUnderCompactionMetrics

Reply:
It's not compaction metrics at all - it's actually the metrics that the temp log and index emit when under compaction.